### ISSUE TITLE
Error in app launcher

### ISSUE THREAD

**Issue by Joao-Queiroga** [BlingCorp/bling#199]
Every time I tried putting the app launcher in the configuretion, awesome gets an error saying that I am tring to call a nil value (field append_client_mousebinding)
![Awesomewm error](https://user-images.githubusercontent.com/72106566/218254573-0d08c184-0a83-4fc3-84cc-6c0709a84707.png)

**Comment by Kasper24** [comment]
You are not using awesome-git, right?

**Comment by Joao-Queiroga** [comment]
I am using awesome 4.3 with luajit on gentoo

**Comment by Kasper24** [comment]
Yeah it's because awesome 4.3 didn't have those methods available, and I think the only way to do so in 4.3 is to set the entire global keys table?
